requirements.tex

```diff
@@ -2,86 +2,89 @@ \section{Stakeholders Requirements} \label{sec:reqs}
 
-The main reason for DM to do software releases is to fulfill the project outcome: provide the Data Products.
+A primary reason to make software releases is provide a mechanism by which the versions of the software used to generate LSST's data products, and hence the provenance of those data products, may be managed and controlled.
 
-The LSST Data Products will be obtained processing the data collected by the telescope using Data Management software products.
-
-In order to controll the provenance of the LSST Data Products, the processing shall be done in a controlled way and using officially released software.
-This imply that the releases shall:
+During the operational era, the Data Management subsystem will have disbanded.
+However, it is important that best practices for release management are established now and can be supplied to the LSST Operations project.
+In particular, the process should deliver software releases which:
 
 \begin{itemize}
-\item fulfill the data processing as per project milestones. This imply:
+\item Enable data processing to be carried out according to the project requirements. This implies that the process should:
   \begin{itemize}
-  \item fulfill the LSST construction requirements as per section \S\ref{sec:nonopsreqs}
-  \item fulfill the operational requirements as per sections \S\ref{sec:procreqs} and \S\ref{sec:otherreqs}
-  \item establish a well controlled release process
-  \item releases to be available as per project milestones.
-  \item releases to include the expected functionalities in order to fulfill the project milestones.
+  \item fulfill the LSST construction requirements as defined in \S\ref{sec:nonopsreqs}
+  \item fulfill the operational requirements as defined in \S\ref{sec:procreqs} and \S\ref{sec:otherreqs}
+  \item deliver documented functionality as described by construction-era milestones and emergent operational requirements.
   \end{itemize}
-\item enable the Science Community to contribute to the project. This imply:
+\item Enable the scientific community to contribute to the project. This imples that the process should:
   \begin{itemize}
-  \item include third-party software from the Science Community when this is beneficial to the project outcome,
-  \item carry an appropriate source code license to permit contributions from the Science Community.
-  \item fulfill the Science Community requirements as per section \S\ref{sec:comreqs}
+  \item fulfill the requirements from the scientific community as defined in \S\ref{sec:comreqs}
+  \item include third-party software provided by the scientific community when this is beneficial to the project outcome
+  \item carry an appropriate source code license.
   \end{itemize}
 \end{itemize}
 
 
-\subsection{Release Requirements for the \gls{LSST} Science Community} \label{sec:comreqs}
+\subsection{Release Requirements for the Interacting with the Scientific Community} \label{sec:comreqs}
```

Comment:
Should be here ".. for Ingreracting with..." ?

Reply:
“Integrating”? I did mean “interacting”, but I think “integrating” would work just about as well, so I've no objection to changing this if you want.

Reply:
Yes.. sorry. I think ".. for the Interacting..." should be just "... for Interacting ..."

Reply:
Oh, you are quite right, of course. :-)